com.unity.mobile.notifications/Runtime/iOS/Plugins/UnityNotificationData.h

```diff
@@ -28,12 +28,12 @@ typedef struct iOSNotificationData
 
     //Custom data
     char* data;
-    BOOL showInForeground;
+    int showInForeground;
     int showInForegroundPresentationOptions;
 
```

Comment:
P.S Something for the future, I think it's a good practice to pass struct size next to the marshalled struct, so you can validate that C# struct size matches native struct size